pytradfri/group.py

```diff
@@ -24,85 +32,116 @@
     ROOT_GROUPS,
 )
 from .error import ColorError
-from .resource import ApiResource
+from .resource import ApiResource, TypeRaw
+
+if TYPE_CHECKING:
+    from .gateway import Gateway
+
+
+class GroupResponse(BaseModel):
```

Comment:
Doesn't the group response have the ApiResourceResponse as base?

Reply:
It does, should it inherit? It does make sense now that you mention it - they are sharing some properties. I'll refactor.